demisto_sdk/commands/content_graph/parsers/json_content_item.py

```diff
@@ -13,8 +12,6 @@
     NotAContentItemException,
 )
 
-logger = logging.getLogger("demisto-sdk")
```

Comment:
in all these files where we remove the import - was it not used? Why was it here in the first place? 🤔

Reply:
It was a mistake. IT wasn't used.